runner.go

```diff
@@ -46,7 +47,8 @@ func newRuntime() *Runtime {
 	}
 	r.flags.StringVar(&configFile, "config", "", "Configuration file to use")
 	r.flags.BoolVar(&nonInteractive, "non-interactive", false, "Non-interactive mode (only applies when running the default set of tasks)")
-	r.flags.BoolVar(&listTasks, "list", false, "List all tasks")
+	r.flags.BoolVar(&listTasks, "list", false, "List all default tasks")
```

Comment:
I don't think it's clear what `default` means. Could we say `List all tasks except those marked "Hidden".`?